docs/getting_started/docker.md

```diff
@@ -0,0 +1,103 @@
+# Running Labelmerge with Docker on Windows
```

Comment:
This is maybe still useful but if we can pip install labelmerge that's probably even easier (at least in some cases)

Reply:
I agree though I would leave the container stuff in to give people different options